### PR TITLE
Bump graphql dependency in adeira__graphql-global-id

### PR DESCRIPTION
**Pull request by michalsanger**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/graphql/graphql-js/releases/tag/v16.0.0

### CODE REVIEW

**Review by mrtnzlml — APPROVED** [review]

**Review by mrtnzlml — APPROVED** [review]

**Review by andrewbranch — CHANGES_REQUESTED** [review]
Why is this needed? The dependency should match the range specified by the implementation package, which is `^14.5.8`. https://unpkg.com/browse/@adeira/graphql-global-id@1.2.0/package.json

**Comment by michalsanger** [comment]
@andrewbranch GraphQL types of v15 and v16 are not compatible so I can't use `@adeira/graphql-global-id` in a codebase with graphql v16. 

`@adeira/graphql-global-id` is compatible with graphql v16 so I only wanted to update types. Do you suggest to release new version of `@adeira/graphql-global-id` with upgraded peer dependency and then follow with this PR?

**Comment by andrewbranch** [comment]
First of all, these typings are for @adeira/graphql-global-id v1.2, so if you’re referring to v2, the version in the header comment needs to be updated. Until that happens, we’re only talking about v1.2.

@adeira/graphql-global-id@1.2.0 claims to be compatible with graphql@^14.5.8 (which does not include v15 or v16) and @adeira/graphql-global-id@2.0.1 claims to be compatible with graphql@^15.0.0 (which does not include v16). Regardless of whether or not this is accurate, the typings should reflect exactly that.

If you’re also a contributor to the implementation package, then yes, I suggest you review your peerDependency ranges and then follow up here 👍

File: types/adeira__graphql-global-id/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "graphql": "^15.4.0"
+        "graphql": "^16.0.0"
```
Comment (andrewbranch) [inline types/adeira__graphql-global-id/package.json:4]:
```suggestion
        "graphql": "^15.0.0 || ^16.0.0"
```

**Review by andrewbranch — CHANGES_REQUESTED** [review]

**Review by andrewbranch — APPROVED** [review]
Thanks!

**Review by mrtnzlml — APPROVED** [review]

**Comment by michalsanger** [comment]
Ready to merge

**Pull request merged**